src/index.js

```diff
@@ -1,14 +1,51 @@
+/* eslint-disable no-unused-vars */
 import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const inputField = document.getElementById('inputField');
+function saveToLocalStorage(data) {
+  let tasks = [];
+  if (localStorage.getItem('tasks')) {
+    tasks = JSON.parse(localStorage.getItem('tasks'));
+  }
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+  tasks.push(data);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+const target = document.getElementById('listContainer');
+
+function displaytasks(task, index) {
+  const content = `
+      <li class ="commonClass">
+      <label class ="checkBoxLabel">
+      <input for ="job${index}" id="inputField" type="checkbox" class ="checkBox">
+      <p id ="job${index}" class ="pInsideLi">${task.description}</p>
+      </label>
+      <button  class="btnAll threeDots">
+       <i class="fa fa-ellipsis-h"></i>
+      </button>
+      </li>
+    `;
+  return content;
+}
```

Comment:
Good job with additional functionality 💯 

However, as this is a basic project, we are still required to fulfill all the project's core requirements. 

According to the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md), please kindly implement the following 🤗👍

- Create an **index.js** file and set an array of some simple *to do tasks* (array of objects). Each task object should contain three keys: 
  1. description [`string`].
  2. completed [`bool`].
  3. index: [`number`].

- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the `index` values for each task.

Reply:
@abdulhamiid 
resolve the requirement.